InnerEye/ML/configs/histo_configs/classification/DeepSMILECrck.py

```diff
@@ -135,12 +135,23 @@ def get_path_to_best_checkpoint(self) -> Path:
         was applied there.
         """
         # absolute path is required for registering the model.
-        return (
-            fixed_paths.repository_root_directory()
-            / self.checkpoint_folder_path
-            / self.best_checkpoint_filename_with_suffix
-        )
-
+        absolute_checkpoint_path = Path(fixed_paths.repository_root_directory(),
+                                        self.checkpoint_folder_path,
+                                        self.best_checkpoint_filename_with_suffix)
+        if absolute_checkpoint_path.is_file():
+            return absolute_checkpoint_path
+
+        absolute_checkpoint_path_parent = Path(fixed_paths.repository_parent_directory(),
+                                    self.checkpoint_folder_path,
+                                    self.best_checkpoint_filename_with_suffix)
+        if absolute_checkpoint_path_parent.is_file():
+            return absolute_checkpoint_path_parent
```

Comment:
Confused. This and the variable above are exactly the same?

Reply:
I have taken this from our other config. https://github.com/microsoft/InnerEye-DeepLearning/blob/fb258d5c9a3ba18565b5a67e7ac1f00127d9ecb9/InnerEye/ML/configs/histo_configs/classification/DeepSMILEPanda.py#L148 It was only to enable this config to work and may not be related to this PR. Will remove for now for another PR